tests/smoke.rs

```diff
@@ -1,42 +1,87 @@
 use executable_path::executable_path;
```

Comment:
`smoke.rs` is probably not the best name anymore. Any better suggestions?

Reply:
Maybe just `tests/integration.rs`?

Reply:
Sounds good.